src/redux/reducers/basket/type.tsx

```diff
@@ -0,0 +1,19 @@
+import { IJewelryItem } from '../jewelry/type';
```

Comment:
Keep creating a file per entity.
Better to keep the following structure:
redux(or store or root-store)/
- features (a folder with all store features)/
--- basket (feature)/
----- index.ts (module export file)
----- reducers.ts
----- types.ts
----- actions.ts
----- selectors.ts
----- state.ts (initial state)
----- sagas.ts (if used)